gpkit/constraints/bounded.py

```diff
@@ -25,12 +24,14 @@ def varkey_bounds(varkeys, lower, upper):
     constraints = []
     for varkey in varkeys:
         variable = Variable(**varkey.descr)
-        units = varkey.units if isinstance(varkey.units, Quantity) else 1
+        if variable.units:
+            variable.hmap.units = None
+            variable.units = None
```

Comment:
it's a Variable, so it's a Monomial, so by removing its units you're essentially dividing the Monomial by its units